merlin_standard_lib/proto/schema_bp.py

```diff
@@ -619,11 +619,14 @@ class FeaturePresenceWithinGroup(betterproto.Message):
 
 @dataclass
 class InfinityNorm(betterproto.Message):
-    """
+    r"""
     Checks that the L-infinity norm is below a certain threshold between the
     two discrete distributions. Since this is applied to a
-    FeatureNameStatistics, it only considers the top k. L_infty(p,q) = max_i
-    |p_i-q_i|
+    FeatureNameStatistics, it only considers the top K.
```

Comment:
Please let me know if the equation is correct: https://nvidia-merlin.github.io/Transformers4Rec/review/pr-474/api/merlin_standard_lib.proto.html#merlin_standard_lib.proto.schema_bp.InfinityNorm